tlatools/org.lamport.tlatools/src/tla2sany/parser/TLAplusParser.java

```diff
@@ -7765,42 +7786,6 @@ final private boolean jj_2_74(int xla) {
     finally { jj_save(73, xla); }
   }
 
-  final private boolean jj_3R_210() {
```

Comment:
This reformatting doesn't belong here.

Reply:
All the jj methods are automatically generated by JavaCC